### PR TITLE
Fix holey bumblebee svg icon

### PR DESCRIPTION
**Pull request by bierik**
Closes https://github.com/4teamwork/opengever.core/issues/2508

Attach `.sketch` source file as well for further changes.

screen shot 2017-02-01 at 16 52 49
screen shot 2017-02-01 at 16 53 01

### CODE REVIEW

**Comment by deiferni** [comment]
I would suggest something that lives in the toplevel repository, e.g. a `data` folder, and then make sure we don't release that one

File: opengever/bumblebee/browser/resources/fallback.svg [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-<g>
-	<path class="st0" d="M157.8,506.9c-19.5-19.5-51.2-19.5-70.7,0c-19.5,19.5-19.5,51.2,0,70.7s51.2,19.5,70.7,0
-		S177.4,526.4,157.8,506.9z M147.4,554.3c1.1,1.1,1.1,2.8,0,3.8l-9,9c-1.1,1.1-2.8,1.1-3.8,0l-12-12l-12,12c-1.1,1.1-2.8,1.1-3.8,0
-		l-9-9c-1.1-1.1-1.1-2.8,0-3.8l12-12l-12-12c-1.1-1.1-1.1-2.8,0-3.8l9-9c1.1-1.1,2.8-1.1,3.8,0l12,12l12-12c1.1-1.1,2.8-1.1,3.8,0
-		l9,9c1.1,1.1,1.1,2.8,0,3.8l-12,12L147.4,554.3z"/>
-</g>
-</svg>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="400px" height="400px" viewBox="0 0 400 400" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 42 (36781) - http://www.bohemiancoding.com/sketch -->
```
Comment (deiferni) [inline opengever/bumblebee/browser/resources/fallback.svg]: i'd drop these comments

**Comment by bierik** [comment]
@deiferni @Rotonen updated

**Comment by lukasgraf** [comment]
`data` as a name (for pretty much anything) is a no-go for me (overly generic). I would suggest `assets`.

**Comment by bierik** [comment]
@lukasgraf updated

**Comment by deiferni** [comment]
muesch no rebase 😉

**Pull request merged**